frontend_lib/src/component/Button/IconButton.styl

```diff
@@ -13,49 +12,56 @@
     opacity 0.65
     cursor not-allowed
 
-  &__dark
+  &-dark
```

Comment:
I don't understand why this change, can you explain it to me?

Reply:
See @Skylsmoi comment: https://github.com/tracim/tracim/pull/4220#discussion_r583566302